mrjob/bin.py

```diff
@@ -844,14 +844,14 @@ def _interpolate_spark_script_path(self, path):
 
     def _spark_cmdenv(self, step_num):
         """Returns a dictionary mapping environment variable to value,
-        including mapping PYSPARK_PYTHON to self._python_bin()
+        including mapping PYSPARK_PYTHON to self._task_python_bin()
         """
         step = self._get_step(step_num)
 
         cmdenv = {}
 
-        if step['type'] in ('spark', 'spark_script'):  # not spark_jar
```

Comment:
Just curious why it used to say # not spark_jar?

Reply:
Well it's probably not needed if you're executing a jar? But we still need our code in this execution.